README.md

```diff
@@ -32,6 +32,7 @@ _WIP_
 * [Part 11 - Work with SQL](data-analysis/11-work-with-sql.ipynb)
 * [Part 12 - Generate data](data-analysis/12-generate-data.ipynb)
 * [Part 13 - The Power of Parallel Processing](data-analysis/13-data-processing.ipynb)
+* [Part 14 - Distributed Computation](data-analysis/14-distributed-computation/notebook.ipynb)
```

Comment:
![MapReduceImage](https://user-images.githubusercontent.com/58446818/123367276-adc9a600-d547-11eb-9a37-5b91ce2eb090.png)

Reply:
Picture for explaining MapReduce ?

Reply:
looks good!